spec/observe_tail_spec.rb

```diff
@@ -21,6 +21,16 @@
     FileUtils.rm_rf(sincedb_path)
   end
 
+  context "when sincedb path is given but ENV[\"HOME\"] or ENV[\"SINCEDB_PATH\"] is not given" do
```

Comment:
in this test we only test that the absense of HOME is tolerated, maybe we should adapt the name of this context?